backend/tests/conftest.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+
+from app import hackathon_backend_api
+
+
+@pytest.fixture
+def client():
+    # app.app.config['TESTING'] = True
+    yield hackathon_backend_api.test_client()
```

Comment:
what does this test do?

Reply:
the fixture sets up the mock client to use for every test. see uses of pytest-fixtures and conftest.py